developerportal/apps/taskqueue/tasks.py

```diff
@@ -14,6 +18,9 @@
 logger = logging.getLogger(__name__)
 
 
+DELAY_BEFORE_WARMING_CDN = 60 * 15  # Allow 15 mins, so the CDN invalidates
```

Comment:
This is a best-guess for now, based on AWS docs pointing at 10-15 mins for invalidation to take place. I'm not super-happy with this approach, so definitely open to bouncing around some alternatives